pages/style-tokens/color/overview.md

```diff
@@ -151,13 +151,13 @@ The following color wheels are a way to visualize the entire [USWDS system palet
 The vivid palette is incomplete. We’ve added vivid variants as we’ve found a project need. This number will continue to grow and we [welcome new suggetions](https://github.com/uswds/uswds/issues) in our Github repo for vivid variants where none currently exist.
 
 {:.bg-white.padding-2.radius-md}
-![standard color wheel]({{ site.baseurl }}/assets/img/style-tokens/uswds-standard-color-wheel.webp)
+![standard color wheel]({{ site.baseurl }}/assets/img/style-tokens/uswds-standard-color-wheel.jpg)
```

Comment:
Removed the `.webp` images since it's not a well-supported format. We could return them when we have a better responsive image solution